### ISSUE TITLE
Successful conduct load times out

### ISSUE THREAD

**Issue by markusjura** [typesafehub/conductr-cli#112]
Sometimes the `conduct load` times out even if the bundle has been successfully loaded. The below terminal snippet shows that first the bundle is loaded and the `Bundle c6ee0b42a4ed2bb8e3c24cf6c80e97e3 waiting to be installed` step times out. After the timeout it is still possible to successfully run the bundle.

```
mj@MacBook-Pro-2:~/workspace$ conduct load /Users/mj/workspace/lagom/samples/chirper/activity-stream-impl/target/bundle/activity-stream-impl-v1-c6ee0b42a4ed2bb8e3c24cf6c80e97e36becbff7eb0892549c69fb86d83d1370.zip
Retrieving bundle...
Bintray credentials loaded from /Users/mj/.bintray/.credentials
Retrieving file:///Users/mj/workspace/lagom/samples/chirper/activity-stream-impl/target/bundle/activity-stream-impl-v1-c6ee0b42a4ed2bb8e3c24cf6c80e97e36becbff7eb0892549c69fb86d83d1370.zip
Loading bundle to ConductR...
Bundle c6ee0b42a4ed2bb8e3c24cf6c80e97e3 waiting to be installed
conduct Error: Timed out: Bundle c6ee0b42a4ed2bb8e3c24cf6c80e97e3 waiting to be installed

mj@MacBook-Pro-2:~/workspace$ conduct run activity-stream-impl
Bundle run request sent.
Bundle c6ee0b42a4ed2bb8e3c24cf6c80e97e3 waiting to reach expected scale 1
Bundle c6ee0b42a4ed2bb8e3c24cf6c80e97e3 has scale 0, expected 1
Bundle c6ee0b42a4ed2bb8e3c24cf6c80e97e3 expected scale 1 is met
Stop bundle with: conduct stop --ip 192.168.99.100 c6ee0b4
Print ConductR info with: conduct info --ip 192.168.99.100
```

**Comment by huntc** [comment]
It would wait if there are no resources available.

It'd be good to try and reproduce this.

**Comment by huntc** [comment]
Is this problem outstanding? I've not seen it...

**Comment by huntc** [comment]
Closing - should be resolved or fixable relating to #214
